spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -69,6 +69,11 @@
 	 */
 	private final CompletionOperations completionOperations;
 
+	/**
+	 * REST Client for runtime operations.
+	 */
+	private final RuntimeOperations runtimeTemplate;
```

Comment:
To be consistent with the variable naming, can we have `runtimeTemplate` to be `runtimeOperations` ?